### ISSUE TITLE
Improve handling of ').' in SA1009 code fix

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2474]
**Input:**

```csharp
await Task.Delay(1000
    ).ConfigureAwait(false);
```

**Expected result:**

SA1009 is reported. The code fix produces the following:

```csharp
await Task.Delay(1000)
    .ConfigureAwait(false);
```

**Actual result:**

The code fix produces the following:

```csharp
await Task.Delay(1000).ConfigureAwait(false);
```
